### ISSUE TITLE
Unwanted is-flex

### ISSUE THREAD

**Issue by kerams** [Fulma/Fulma#291]
Hello, @MangelMaxime.

Why is the `is-flex` class added here?

https://github.com/Fulma/Fulma/blob/837eefb9a68e7ce908f5dab9c4b16d5722dd90fd/src/Fulma/Common.fs#L638-L652

I'm no CSS guru, but these appear to be properties of [flex items](https://www.w3schools.com/css/css3_flexbox_items.asp). Using any of these 3 modifiers forces the element to be a flexbox too, which is not desirable. Can the extra class be removed?

**Comment by MangelMaxime** [comment]
Hello @kerams,

Hum, I think this is indeed a mistake on my part. When implement the flex modules I saw stuff like `FlexAlignItems`, `FlexAlignContent`, which requires to be on a `display: flex` element to works. So I tough let's just auto add `is-flex` to like Fulma does in some others places.

And I think I just made a big multi edit selection without thinking much further.

So after checking, it seems like, the following does needed the `IsFlex`:

- FlexDirection
- FlexWrap
- FlexJustifyContent
- FlexAlignContent
- FlexAlignItems

but not:

- FlexAlignSelf 
- FlexGrow 
- FlexShrink 

I will remove the extra class.
